Remotion/Web/Core/UI/Controls/DatePickerPage.cs

```diff
@@ -108,6 +136,16 @@ override protected void OnInit (EventArgs e)
     //  Force the creation of the postback function
     Page.ClientScript.GetPostBackEventReference(this, "");
 
+#if !NETFRAMEWORK
+    var globalizationService = SafeServiceLocator.Current.GetInstance<IGlobalizationService>();
+    var resourceManager = globalizationService.GetResourceManager(typeof(ResourceIdentifier));
+    Calendar.TitleText = resourceManager.GetString(ResourceIdentifier.TitleText);
+    Calendar.SelectMonthTitle = resourceManager.GetString(ResourceIdentifier.SelectMonthTitle);
+    Calendar.SelectWeekTitle = resourceManager.GetString(ResourceIdentifier.SelectWeekTitle);
```

Comment:
Are these two actually in use in re-motion? If not, we should drop the resource entries to remove the maintenance overhead.

You can test the DatePicker via
http://localhost:60500/res/Remotion.Web/Themes/NovaGray/UI/DatePickerForm.aspx?Culture=en-US&UICulture=de-AT

Reply:
We don't have any usages of `SelectionMode` which would enable those two buttons but it would be possible to override the `DatePickerPage` and change that? I don't see the maintenance overhead tbh. We already have the localizations and the logic/localizations won't change

Yes primitives is going to be added in a new PR+issue

Reply:
No, there's no override to enable this feature. Maintenance overhead here means that all strings are exported and get reviews and are re-evaluated for correctness or simply require translation if there's ever another language added. If we have completely unused/unneeded strings, all this adds to the total cost of ownership down the road.